frontend/app/components/themeProvider.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { ReactNode } from "react"
+import theme from "@/styles/theme"
+import { ThemeProvider as MuiThemeProvider } from "@mui/material/styles"
+
+interface ThemeProviderProps {
+  children: ReactNode
+}
+
+const ThemeProvider: React.FC<ThemeProviderProps> = ({
+  children,
+}): React.ReactElement => {
+  return <MuiThemeProvider theme={theme}>{children}</MuiThemeProvider>
+}
+
+export default ThemeProvider
```

Comment:
Named export here. Let's prefer named export over default exports in our code base.